easy-coding-standard.neon

```diff
@@ -0,0 +1,130 @@
+includes:
+    # pick set from https://github.com/Symplify/EasyCodingStandard/tree/master/config
+    # for PSR-2 description, see: http://www.php-fig.org/psr/psr-2/
+    - vendor/symplify/easy-coding-standard/config/psr2.neon
+
+checkers:
+    # Shopsys Checkers
+    - ShopSys\CodingStandards\Sniffs\ForbiddenExitSniff
+    - ShopSys\CodingStandards\Sniffs\ForbiddenSuperGlobalSniff
+    - ShopSys\CodingStandards\CsFixer\MissingButtonTypeFixer
+    - ShopSys\CodingStandards\CsFixer\OrmJoinColumnRequireNullableFixer
+
+    - PHP_CodeSniffer\Standards\PSR1\Sniffs\Methods\CamelCapsMethodNameSniff
+
+    # Fine-tuning of PSR-2 standard
+    # Sometimes, there is no other way than have longer lines. For example when using fully-qualified namespaces in type hinting comments.
+    PHP_CodeSniffer\Standards\Generic\Sniffs\Files\LineLengthSniff:
```

Comment:
I am still not sure about this sniff, the psr-2 says: "There MUST NOT be a hard limit on line length". I think the best solution would be to omit the rule completely.

Reply:
Absolutely fine with me. 

I'm slowly moving to "automatically [fix only valid cases](https://github.com/Symplify/CodingStandard#arguments-should-be-on-the-samestandalone-line-to-fit-line-length) and do not annoy user about it" approach

I'll handle that in a minute